Formula/gucharmap.rb

```diff
@@ -27,7 +27,8 @@ def install
                           "--prefix=#{prefix}",
                           "--disable-Bsymbolic",
                           "--disable-schemas-compile",
-                          "--enable-introspection=no"
+                          "--enable-introspection=no",
+                          "--with-unicode-data=download"
```

Comment:
So we're all on the same page, this is what it does

gucharmap/unicode.sha512sums
```
SHA512 (Blocks.txt) = b841d4274de846d6931db8a4cc8fa5ec1a6b72e43b0693b53376015b121a38ca921ef1bf67dcc827ce93822d48806ec3afd4864fc123c5c4163c6372585377d7
SHA512 (DerivedAge.txt) = d2c0cfaccd936255c3c04907b16e74cc992c70a35283885e1e35c36bebe7199c4c50813309f5ffbfcd09b979c0201cc3953a40d0b2faad485b670d68700e0945
SHA512 (NamesList.txt) = 8651b522364b999f15052a2f1585a356f69a0ca69f248a75b7b610a163d45a84fc5eb8146c7a9ea72c913a160a9f2276a0f6dd06a72e3b60951a06f9843380a0
SHA512 (Scripts.txt) = 9c5e17c4731c1d9fb5e733b81b49ae815b36a390f6c3f02c0c4637e62dc59dde8e503956fa4378f85c215af2effad7907e846f49c514e640f494e5baf8b46ae6
SHA512 (UnicodeData.txt) = 43eaf66d9cb3748012b2dfd77da1b41f667c5c7602a56bea8186b796b215bde82d555d79ab053378c2222521396354dcce5cf23a78fa3b1456062c47771c8433
SHA512 (Unihan.zip) = 9856d30afe5d7b0b088bde36c12431eae267992b315944ca8a7ddb3559adee7474fedd2d0d6b22716439f9cddad573a4bb45c6209036b0066a85fbd19412852e
```

```
download-unicode-data:
        for f in $(unicode_files); do \
                [ -e $(builddir)/$${f} ] || $(WGET) $(WGETFLAGS) "http://www.unicode.org/Public/$(unicode_version)/ucd/$${f}"; \
        done && \
        $(SHA512SUM) --check $(srcdir)/unicode.sha512sums

unicode.sha512sums: stamp-download-unicode-data
        $(AM_V_GEN)$(SHA512SUM) --tag $(addprefix $(unicodedatadir)/,$(unicode_files)) | $(SED) 's,$(unicodedatadir)/,,g' > $(srcdir)/$(@F)

stamp-unicode-data: stamp-download-unicode-data
        PROG_UNZIP=$(PROG_UNZIP) $(srcdir)/gen-guch-unicode-tables.pl $(unicode_version) $(unicodedatadir) && \
        echo timestamp > $(@F)
```